### ISSUE TITLE
Import-Export tool (and Proton-Bridge) is messing with Content-Disposition of parts

### ISSUE THREAD

**Issue by exander77** [ProtonMail/proton-bridge#99]
By this code in `pkg/message/header.go`:
```
disposition := "attachment" //nolint[goconst]
  if strings.Contains(att.Header.Get("Content-Disposition"), "inline") {
    disposition = "inline"
  }
```

All parts are set to `attachment` if they are not `inline`:

`Content-Disposition` can be completely missing. In that case, it should not be set.

The similar problem is on the server (when receiving email), it sets `Content-Disposition: inline` instead.

**Comment by andrzejsza** [comment]
thanks for reporting this. it unraveled an issues with the API, which was not recognising `Content-Disposition` in a way that Bridge was expecting it to. We're looking into this.

**Comment by andrzejsza** [comment]
this has been fixed a while back in 1.6.9. closing.
